docs/architecture/devops-for-aspnet-developers/actions-build.md

```diff
@@ -0,0 +1,197 @@
+---
+title: DevOps with .NET and GitHub Actions - Build a .NET Web App
+description: Start your journey of DevOps with .NET and GitHub Actions by building a .NET web app
+author: colindembovsky
+ms.date: 03/04/2021
+---
+# Build a .NET web app using GitHub Actions
+
+[GitHub Actions](https://github.com/features/actions) allow you to automate workflows in response to events that are triggered in GitHub. A common workflow is Continuous Integration (CI), but Actions can automate other processes. For example, sending welcome emails when people join a repository.
+
+To explore moving code to the cloud, you'll build a GitHub Actions workflow file. The workflow file will be used for the Simple Feed Reader app you've already deployed to Azure App Service.
+
+In this article, you will:
+> [!div class="checklist"]
+
+> * Learn the basic structure of a GitHub Action workflow YAML file.
+> * Use a template to create a basic build workflow that builds the .NET app and executes unit tests.
+> * Publish the compiled app so that it's ready for deployment.
```

Comment:
I doubt this will render correctly on docs site without a `>` in the empty line (line 15)
But please confirm on staging first before taking the suggestion:

```suggestion

> [!div class="checklist"]
>
> * Learn the basic structure of a GitHub Action workflow YAML file.
> * Use a template to create a basic build workflow that builds the .NET app and executes unit tests.
> * Publish the compiled app so that it's ready for deployment.
```

Reply:
It's rendering fine 

![image](https://user-images.githubusercontent.com/13281246/114818015-a3b55980-9dd8-11eb-8f79-ed74821edd14.png)

Reply:
It should render with check marks, not a quote. Similar to what's in https://docs.microsoft.com/en-us/dotnet/framework/wcf/how-to-implement-a-wcf-contract.

(Note that this is not the only occurrence of a missing `>`. So it might be easier for you to review locally and push)

Reply:
oh ok. Now, I get it. Yes, I will make that change locally.

Reply:
![image](https://user-images.githubusercontent.com/13281246/114835995-ce5edc80-9def-11eb-883f-ec492a53b475.png)